CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## TBD
+### Changed
+- `MessageReader.read()` now returns `MessageEvent`, wrapping `ListenerEvent` and `MessageProperty`.
```

Comment:
I guess this means a major version change right, API has changed.

Reply:
Yes, will change this to `## [2.0.0] - TBD`

Reply:
ok can you bump the versions of the pom already so we won't forget if this PR for some reason isn't merged for while:
`mvn versions:set -DnewVersion=2.0.0-SNAPSHOT -DgenerateBackupPoms=false`

Reply:
Done, thanks